### PR TITLE
remove kyma 1 related resources

### PR DESCRIPTION
**Pull request by strekm**
**Description**

Changes proposed in this pull request:

- removed istio-installer
- remove installation/certs
- remove installation/scripts
- passing CODEOWNERS of installation folder to @tobiscr 

**Related issue(s)**

### CODE REVIEW

**Review by mjakobczyk — DISMISSED** [review]

File: installation/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
-# Kyma Installer
+# Installation Resources
 
 ## Overview
 
-The Kyma Installer is a proprietary solution based on the Kubernetes operator. See the document on [local Kyma installation](https://kyma-project-old.netlify.app/docs/#installation-install-kyma-locally) for basic information on how to use the installer and run Kyma locally. Read also the [installation instructions](https://kyma-project-old.netlify.app/docs/#installation-install-kyma-on-a-cluster) to learn how to set up Kyma on a cluster.
+This folder contains Kyma resources CRDs and components list for installation.
```
Comment (majakurcius) [inline installation/README.md:5]:
```suggestion
This folder contains Kyma custom resource definitions (CRDs) and the components list for installation. 
```

**Review by majakurcius — APPROVED** [review]

**Review by dennis-ge — APPROVED** [review]
Also wanted to remove some kyma 1 related stuff in https://github.com/kyma-project/kyma/pull/13578. Closed my PR since it is a duplicate of this one :)

**Review by a-thaler — APPROVED** [review]

**Review by mjakobczyk — APPROVED** [review]

**Pull request merged**